steps/id-and-residency/present-for-check.tsx

```diff
@@ -35,6 +35,7 @@ const TenantsSelect: React.FunctionComponent<Omit<
   return (
     <Checkboxes
       {...props}
+      required={true}
```

Comment:
```suggestion
      required
```

This the same in React.

Reply:
Also, this change has snuck in from somewhere else. In future, please try to isolate your work to the ticket at hand.